eng/pipelines/templates/steps/build-test-go.yaml

```diff
@@ -129,12 +128,3 @@ steps:
       testRunner: JUnit
       testResultsFiles: ${{ parameters.GoTestPath }}/**/report.xml
       failTaskOnFailedTests: true
-
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(parameters.PublishAutorestCoverage, true))}}:
```

Comment:
Is this no longer supported?

Reply:
Correct do to security we cannot use SAS tokens any longer and instead of updating the tooling we decided to stop publishing coverage.

Reply:
> Correct do to security we cannot use SAS tokens any longer and instead of updating the tooling we decided to stop publishing coverage.

How are we tracking the CADL code coverage now? What is the plan to bring back support for this?
//cc @lmazuel, @allenjzhang

Reply:
The cadl ranch code coverage is still happening this is only the autorest testserver coverage.

Reply:
What's the plan for bringing this coverage tracking back? Is there an issue tracking that work?

Reply:
No we are retiring the publishing of the coverage reports for autorest. It still exists with cadl-ranch for the newer typespec code gen.